src/graphql/getType.js

```diff
@@ -102,6 +103,17 @@ const getType = memoize(type => {
   if (type.isDomain)
     return getType(type.baseType)
 
+  if (type.isComposite) {
+    return new GraphQLInputObjectType({
+      name: upperFirst(camelCase(type.name)),
```

Comment:
Normally, I like to put the name conversions in `catalog.js`. See [here](https://github.com/calebmer/postgraphql/blob/master/src/postgres/catalog.js#L167-L173). This also has the advantage of being able to leverage [these two](https://github.com/calebmer/postgraphql/blob/master/src/postgres/catalog.js#L8-L13) functions which allow us to preserve leading and trailing underscores. So `__foo_bar__` would become `__fooBar__` and not `fooBar`.